### PR TITLE
Fix/dr docs 1

### PR DESCRIPTION
**Pull request by davidrhodus**
Add CONTRIBUTING.md and a few minor README changes

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+git remote add upstream https://github.com/Linnia-Smart-Contracts/Linnia-Smart-Contracts.git
+git fetch upstream
+git pull --rebase upstream master
+```
+NOTE: The directory `Linnia-Smart-Contracts` represents your fork's local copy.
+
+2) Branch out from `master` into `fix/some-bug-#123`:
+(Postfixing #123 will associate your PR with the issue #123 and make everyone's life easier =D)
+```
+git checkout -b fix/some-bug-#123
```
Comment (MaxBlaushild) [inline CONTRIBUTING.md]: I think we decided on 'lp123/bug/fixesTheThing'

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
+Contributing to Linnia-Smart-Contracts
+=======
+
+We really appreciate and value contributions to Linnia-Smart-Contracts. Please take 5' to review the items listed below to make sure that your contributions are merged as soon as possible.
```
Comment (godfreyhobbs) [inline CONTRIBUTING.md]: `5 minutes`  

5'  maybe be confusing for folks learning English.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+## Creating Pull Requests (PRs)
+
+As a contributor, you are expected to fork this repository, work on your own fork and then submit pull requests. The pull requests will be reviewed and eventually merged into the main repo. See ["Fork-a-Repo"](https://help.github.com/articles/fork-a-repo/) for how this works.
+
+## A typical workflow
+
+1) Make sure your fork is up to date with the main repository:
+
+```
+cd Linnia-Smart-Contracts
```
Comment (godfreyhobbs) [inline CONTRIBUTING.md]: Can we just point to github's docs?

File: CONTRIBUTING.md (hunk 2 above)
Comment (tesla809) [reply CONTRIBUTING.md]: Could be just "Please take time to review the items listed below to make sure that your contributions are merged as soon as possible."

**Pull request merged**